packages/edit-site/src/store/actions.js

```diff
@@ -441,3 +442,31 @@ export function* revertTemplate( template ) {
 		);
 	}
 }
+/**
+ * Returns an action object used in signalling that the user opened an editor sidebar.
+ *
+ * @param {?string} name Sidebar name to be opened.
+ *
+ * @yield {Object} Action object.
+ */
+export function* openGeneralSidebar( name ) {
```

Comment:
`NB:`

Does this need to be handled as a control effect because it's dispatching to a separate store?

Reply:
Yeah, we use controls.dispatch to send this over to the interfaceStore as is done with the equivalent actions in the edit-post store.  Or were you proposing a different approach here?